virttest/utils_conn.py

```diff
@@ -2335,3 +2335,24 @@ def conn_setup(self):
         self.connect_to_unix_socket(tcp2unix_session, dest_pmsocat_path)
         self.add_firewall_ports(tcp2unix_session)
         LOG.debug("TCP2UNIX setup successfully.")
+
+
+def update_crypto_policy(value="DEFAULT", timeout=None, verbose=True, ignore_status=True, shell=True):
+    """
+    Update the crypto policy, see "man update-crypto-policies".
+
+    :param value: the policy value, see "man crypto-policies".
+    :param timeout: Time limit in seconds before attempting to kill the process
+    :param verbose: Whether to log the command run and stdout/stderr.
+    :param ignore_status: Whether to raise an exception when command returns
+                          =! 0 (False), or not (True).
+    :param shell: Whether to run the command on a subshell
+    """
+    cmd = "update-crypto-policies --set %s" % value
+    cmd_res = process.run(cmd, timeout, verbose, ignore_status, shell)
```

Comment:
Can you update to use keyword arguments? 
you know, we are still using avocado 82lts and this change does not work for it.
An error occurred in my environment - ERROR: Invalid value (True) set in allow_output_check (29.01 s)

Reply:
Thanks, updated.